governance/polls/Adjust the WBTC Debt Ceiling - July 27, 2020

```diff
@@ -0,0 +1,35 @@
+---
+title: Adjust the WBTC Debt Ceiling - July 27, 2020
+summary: Signal your support for increasing the WBTC Debt Ceiling from 20MM to 34MM.
+discussion_link: https://forum.makerdao.com/t/signal-request-should-we-raise-wbtc-debt-ceiling/3223?u=andy_mccall
+vote_type: Plurality Voting
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Adjust the WBTC Debt Ceiling - July 27, 2020
+
+The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the Maker Governance Community. The community can use this poll express support or opposition to increasing the WBTC Debt Ceiling from 20MM to 34MM.
+
+This Governance Poll ([FAQ](https://community-development.makerdao.com/governance/governance#is-there-more-than-one-type-of-vote)) will be active for three days beginning on Monday, July 27 at 5 PM UTC, the results of which may inform an Executive Vote ([FAQ](https://community-development.makerdao.com/governance/governance#what-is-continuous-approval-voting)) which will go live on Friday, July 31, at 5 PM UTC.
+
+## Review
+
+Additional context was presented in a [Signal Request](https://forum.makerdao.com/t/signal-request-should-we-raise-wbtc-debt-ceiling/3223?u=andy_mccall) on July 15th, 2020. Please review the discussion there to inform your position before voting.
+
+## Next Steps
+
+On the Friday following the conclusion of the poll, there will be an Executive Vote asking MKR token holders if they support or reject the change proposed by this Governance Poll.
+
+---
+
+## Resources
```

Comment:
I've been replacing this older version of the resources with a more up to date version which can be found in the templates I mentioned, or it can be copied from to TUSD PR here: https://github.com/makerdao/community/pull/689/files

Could you use the up-to-date one please? :)

Reply:
👍